R/type-list-of.R

```diff
@@ -38,17 +30,18 @@ as_list_of <- function(x, ...) {
 }
 
 #' @export
-as_list_of.vctrs_list_of <- function(x, .ptype = NULL, ...) {
+as_list_of.vctrs_list_of <- function(x, ..., .ptype = NULL) {
```

Comment:
I think this `.ptype` arg was in the wrong spot. `as_list_of.list()` has it after the `...` and I think that is right. We can change back if revdeps show that it breaks anything but this feels better to me